easybuild/easyconfigs/c/Cython/Cython-0.27.3-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,54 @@
+##
+# EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2013 The Cyprus Institute
+# Authors::   Andreas Panteli <a.panteli@cyi.ac.cy>,
+#             Thekla Loizou <t.loizou@cyi.ac.cy>,
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# License::   MIT/GPL
+#
+# Updated: Pavel Grochal (INUITS)
+##
+easyblock = 'PythonPackage'
+
+name = 'Cython'
+version = '0.27.3'
+
+homepage = 'https://cython.org/'
+description = """
+Cython is an optimising static compiler for both the Python programming
+language and the extended Cython programming language (based on Pyrex).
+"""
+docurls = [
+    "http://cython.org/#documentation",
+    "https://github.com/cython/cython",
+]
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['6a00512de1f2e3ce66ba35c5420babaef1fe2d9c43a8faab4080b0dbcc26bc64']
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+download_dep_fail = True
+
+use_pip = True
+
+modextrapaths = {
+    'PYTHONPATH': ["vars", "lib/python%(pyshortver)s/site-packages"],
```

Comment:
Not needed when using `PythonPackage`, and incorrect when using `multi_deps` since this will hardwire it to a specific Python version.

Reply:
I overlooked the `vars` entry here, why is this needed?

There doesn't seem to be a `vars` subdirectory at all in the `Cython` installation directory?